pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -318,6 +331,11 @@ func (ca *changeAggregator) Start(ctx context.Context) {
 	ctx = logtags.AddTag(ctx, changeAggregatorLogTag, nil /* value */)
 	ctx = ca.StartInternal(ctx, changeAggregatorProcName)
```

Comment:
i believe you need to pass the aggregator to StartInternal, similar to other bulk jobs.